media/pom.xml

```diff
@@ -57,6 +57,14 @@
             <artifactId>common-library</artifactId>
             <version>${revision}</version>
         </dependency>
+        <dependency>
+            <groupId>com.yas</groupId>
+            <artifactId>common-library</artifactId>
+            <version>${revision}</version>
+            <classifier>tests</classifier>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Should we move common-library dependency to parent pom.xml?

Reply:
I attempted to move the `common-library` dependency to the parent `pom.xml`, but encountered a circular dependency error. This issue arises because the version of `common-library` is declared to depend on the version of `yas`. Adding the common-library dependency to the `yas`  `pom.xml` would cause the common-library to reference itself during the Maven build process.
This is the common-library `pom.xml`
```
  <parent>
      <groupId>com.yas</groupId>
      <artifactId>yas</artifactId>
      <version>${revision}</version>
      <relativePath>../pom.xml</relativePath>
  </parent>
  <artifactId>common-library</artifactId>
  <version>${revision}</version>
  <packaging>jar</packaging>
```
And this is yas `pom.xml`:

![image](https://github.com/user-attachments/assets/204945f4-0213-4ab4-ad9d-9d24e5f43d2e)

And this is the error:
![image](https://github.com/user-attachments/assets/5a193862-f56b-44b9-bb0c-5bfa77f7527d)

I have not been able to find a solution to this problem. Could you please review the current implementation? Thank you


> Should we move common-library dependency to parent pom.xml?